spec/models/green_lanes/identified_measure_type_category_assessment_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe GreenLanes::IdentifiedMeasureTypeCategoryAssessment do
+  describe 'attributes' do
+    it { is_expected.to respond_to :measure_type_id }
+    it { is_expected.to respond_to :theme_id }
+    it { is_expected.to respond_to :created_at }
+    it { is_expected.to respond_to :updated_at }
+  end
+
+  describe 'validations' do
+    subject(:errors) { instance.tap(&:valid?).errors }
+
+    let(:instance) { described_class.new }
+
+    it { is_expected.to include measure_type_id: ['is not present'] }
+    it { is_expected.to include theme_id: ['is not present'] }
+  end
+
+  describe 'validations duplicates' do
```

Comment:
It feels like we're using a describe here as a context (context is a thing that varies). Essentially you're describing validations and varying the context of what validations get triggered https://www.betterspecs.org/ but essentially describing the same thing. In fact from a SUT perspective the thing you're really describing is `#valid?`

Reply:
Changed it to a context